clearwater-socket-factory/clearwater-socket-factory-mgmt.conf

```diff
@@ -0,0 +1,23 @@
+# clearwater-socket-factory-mgmt
+#
+# This process runs as root in the management namespace, and enables processes
+# running in other namespaces to obtain sockets in this namespace.
+
+description "Clearwater management socket factory server"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+
+script
+  WHITELIST_DIR=/etc/clearwater-socket-factory/management.d
+  NAMESPACE=management
+
+  . /usr/share/clearwater/bin/clearwater-socket-factory-common
+
+  # Sleep for 5 seconds, to avoid upstart respawning socket-factory too quickly
+  sleep 5
```

Comment:
Instead of putting this in the script body, you can use `post-stop exec sleep 5` (see [here](https://github.com/Metaswitch/clearwater-licensing-support/blob/master/debian/clearwater-tokennet.upstart#L10) for an example).